### PR TITLE
Linux fix, contribution update, ask rework

### PR DESCRIPTION
**Pull request by LEQUI000**
please please please check to see I didn't break anything

### CODE REVIEW

File: cogs/general.py [hunk 1]
```diff
@@ -551,17 +551,27 @@ async def linux(self, ctx, user: typing.Optional[discord.Member] = None):
...
         text = """You can download the Android app here: <https://play.google.com/store/apps/details?id=com.blade.shadowcloudgaming&hl=en>
 You can join the Android app beta here: <https://play.google.com/apps/testing/com.blade.shadowcloudgaming>"""
         await self.bot.general.text_command_process(ctx=ctx, user=user, text=text, command_name="android")
 
-    @commands.command(aliases=['ask'])
-    async def _ask(self, ctx, user: typing.Optional[discord.Member] = None):
-        text = "https://www.dontasktoask.com/"
+    @commands.command(aliases=['_ask'])
+    async def ask(self, ctx, user: typing.Optional[discord.Member] = None):
+        text = """https://dontasktoaskjustask.com/"""
```
Comment (LEQUI000) [inline cogs/general.py:566]: wrong domain

Comment (LEQUI000) [reply cogs/general.py]: adjusted in latest commit

File: cogs/general.py [hunk 2]
```diff
@@ -524,8 +524,8 @@ async def win32(self, ctx, user: typing.Optional[discord.Member] = None):
 Alpha: Accessible at <#593516344415354880>"""
         await self.bot.general.text_command_process(ctx=ctx, user=user, text=text, command_name="win32")
 
-    @commands.command(aliases=['mac', 'macos'])
-    async def win32(self, ctx, user: typing.Optional[discord.Member] = None):
```
Comment (stavlor) [inline cogs/general.py:528]: What What?!?!?!

File: cogs/general.py [hunk 3]
```diff
@@ -551,17 +551,23 @@ async def linux(self, ctx, user: typing.Optional[discord.Member] = None):
...
+    @commands.command(aliases=['pixel'])
     async def android(self, ctx, user: typing.Optional[discord.Member] = None):
         text = """You can download the Android app here: <https://play.google.com/store/apps/details?id=com.blade.shadowcloudgaming&hl=en>
 You can join the Android app beta here: <https://play.google.com/apps/testing/com.blade.shadowcloudgaming>"""
         await self.bot.general.text_command_process(ctx=ctx, user=user, text=text, command_name="android")
 
-    @commands.command(aliases=['ask'])
-    async def _ask(self, ctx, user: typing.Optional[discord.Member] = None):
-        text = "https://www.dontasktoask.com/"
+    @commands.command(aliases=['_ask'])
```
Comment (stavlor) [inline cogs/general.py:560]: _ask shouldn't be needed in this case leave aliases=[] instead

**Pull request merged**